docs/triggers.md

```diff
@@ -24,7 +25,7 @@ the following fields:
     `Trigger` resource object, for example a `name`.
   - [`spec`][kubernetes-overview] - Specifies the configuration information for
     your Trigger resource object. The spec include:
-    - [`bindings`] -  A list of `TriggerBindings` reference to use or embedded TriggerBindingsSpecs to use
+    - [`bindings`] - (Optional) A list of bindings to use. Can either be a reference to existing `TriggerBinding` resources or embedded name/value pairs.
```

Comment:
Let's also update `docs/triggerbindings.md`.

Reply:
`docs/triggerbindings.md` right now contains sections just on the TriggerBinding type and its syntax. The sections on embedded bindings are in the docs/triggers.md and docs/eventlisteners.md